ribs/emitters/opt/_gradient_ascent_opt.py

```diff
@@ -0,0 +1,34 @@
+"""Provides GradientAscentOpt.
+
+Adapted from:
+https://github.com/icaros-usc/dqd/blob/main/ribs/emitters/opt/_adam.py
+https://github.com/hardmaru/estool/blob/master/es.py
+"""
+import numpy as np
+
+from ribs.emitters.opt._gradient_opt_base import GradientOptBase
+
+
+class GradientAscentOpt(GradientOptBase):
+    """Vanilla gradient ascent.
+
+    Args:
+        theta0 (array-like): 1D initial solution.
+        step_size (float): Used to scale the gradient during the update.
+    """
+
+    def __init__(self, theta0, step_size):
```

Comment:
can we have a default for step_size like adam_opt?

Reply:
resolved offline - gradient ascent usually does not have a default step_size as in pytorch